### DISCUSSION TITLE
How to search in embeddings

### DISCUSSION THREAD

**Discussion by EliasPereirah** [openai-php/client#136] [Q&A] [answered]
Sorry I'm starting on this.
After creating the embeddings, how can I search them?

**Comment by pb30** [comment, marked as answer]
You're responsible for storing them in your own vector database (Pinecone, Weaviate, ChromaDB, Postgres with pg_vector, etc), along with the some form of document IDs or the raw text content of the chunk. OpenAI (and this library) doesn't handle any of the storage/searching of embeddings, just the generation.

**Reply by EliasPereirah** [reply]
Thanks for the answer, now I know what to research/study.
